driver-adapters/planetscale-node-basic/index.js

```diff
@@ -0,0 +1,152 @@
+const { Prisma, PrismaClient } = require('@prisma/client')
+const { createPlanetScaleConnector } = require('@jkomyno/prisma-planetscale-js-connector')
+
+const connectionString = process.env.DRIVER_ADAPTERS_PLANETSCALE_NODE_BASIC_DATABASE_URL
+
+const jsConnector = createPlanetScaleConnector({
+  url: connectionString,
+})
+
+const prisma = new PrismaClient({ jsConnector })
+
+exports.handler = async () => {
+  const getResult = async (prisma) => ({
+    prismaVersion: Prisma.prismaVersion.client,
+    deleteMany: await prisma.user.deleteMany().then(() => ({ count: 0})),
```

Comment:
I find it weird 🤔 
I guess you wanted to not have different numbers because of concurrency sometimes?

Reply:
If the previous test fails for any reason, you could have a different number, so juts playing it safe so it does not error more.